templates/greenlight.nginx.j2

```diff
@@ -3,7 +3,7 @@ limit_req_zone $binary_remote_addr zone=req_zone:10m rate=30r/s;
 limit_conn_zone $binary_remote_addr zone=conn_zone:10m;
 
 server {
-  server_name  {{ ansible_host }} {{ greenlight_aliases|join(' ') }};
+  server_name {{ ansible_host }} {{ greenlight_aliases|join(' ') }};
```

Comment:
Shouldn't this be `greenlight_endpoint_name`? I think that's what was causing an issue on my end, since `ansible_host` is not necessarily the hostname of the greenlight endpoint. There is the same problem line 14.

Reply:
If you tested this with another solution and it works for you please suggest a change.

Reply:
I don't see the difference between greenlight_hostname that you added to setup the https certificates and the greenlight_endpoint_name that was already used in env.j2 I think these two are the same and refer to the hostname of the greenlight endpoint. I'll make an issue and a pull request later for that and add that same hostname to the nginx config.